.travis.yml

```diff
@@ -13,13 +13,23 @@ matrix:
   include:
     - os: linux
       env: CUDA=8.0.61-1
-    - os: osx
+    - os: linux
+      env: QT_BASE=59
     - os: linux
       env: STATIC_ANALYSIS=ON
+    - os: osx
+      env: QT_BASE=latest
 
 before_install:
-   - source script/travis/before_script.sh
+  - source script/travis/before_script.sh
```

Comment:
Shall we then move this into `install` too?

Reply:
Sure. I have to admit that I don't really know the difference between those two. I will check the documentation.

Reply:
I think I made mistake in first place (long time ago) which is not critical. `before_install` should be something like preparation step before installation. I could be wrong of course :)